src/types/redis_bloom_chain.cc

```diff
@@ -105,9 +106,14 @@ rocksdb::Status BloomChain::bloomCheck(const Slice &bf_key, const std::string &i
   BlockSplitBloomFilter block_split_bloom_filter;
   block_split_bloom_filter.Init(std::move(bf_data));
 
-  uint64_t h = BlockSplitBloomFilter::Hash(item.data(), item.size());
-  *exist = block_split_bloom_filter.FindHash(h);
-
+  for (size_t i = 0; i < items.size(); ++i) {
+    if (exists->at(i)) {
```

Comment:
Why? this is pre-checked before calling `blockCheck`?

Reply:
I think if the item exists in other sub-bloomfilter already, It's not necessary to check in this sub-bloomfilter.

Reply:
Oh I see, you can add a comment for this